IntegrationTests/UI.IntegrationTests/CommandsDialogs/FileStatusListTests.cs

```diff
@@ -48,7 +48,7 @@ public void ItemSelections()
             var itemAt0 = item1;
             var itemAt1 = item2;
             var itemAt2 = item0;
-            _fileStatusList.SetDiffs(items: items);
+            _fileStatusList.SetDiffs(null, null, items: items);
```

Comment:
Yes, Re# injects these -- but it slows down the IDE too much for me.
```suggestion
            _fileStatusList.SetDiffs(selectedRev: null, parentRev: null, items);
```

Reply:
ok, for all